### ISSUE TITLE
Support allowAdd and allowRemove inside SimpleFormIterator or ArrayInput.

### ISSUE THREAD

**Issue by ashim-saha** [marmelab/react-admin#1805]
We are migrating from `admin-on-rest@v1.4.1` to `react-admin@2.0.0-RC2`.

Previously in `v1.4.1` we use [aor-embedded-array](https://github.com/MhdSyrwan/aor-embedded-array) with `allowAdd` and `allowRemove` options.

Similar kind of functionality is available in `react-admin@2.0.0-RC2` also, ie

```
<ArrayInput source="backlinks">
    <SimpleFormIterator>
        <DateInput source="date" />
        <UrlInput source="url" />
    </SimpleFormIterator>
</ArrayInput>
```

But it does not have `allowAdd` and `allowRemove` options inside `SimpleFormIterator` or `ArrayInput`.

It will be great if we have that.

**Comment by ashim-saha** [comment]
The visibility of `ADD` and the `REMOVE` button are control by `allowAdd` & `allowRemove` options.

For example if we pass `allowRemove={false}` and `allowAdd={false}` then `ADD` and `REMOVE` button are not visible.
```
<EmbeddedArrayInput source="links" allowRemove={false} allowAdd={false}>
     <LongTextInput source="url" />
     <LongTextInput source="context" />
     <EmbeddedArrayInput source="metadata">
         <TextInput source="name" />
         <TextInput source="value" />
     </EmbeddedArrayInput>
 </EmbeddedArrayInput>
```

**Comment by muticansel** [comment]
Is this feature available now? Because when I update the react-admin package in my project there is no allowAdd or allowRemove feature. I'm doing something wrong?

**Comment by ashim-saha** [comment]
This feature is not released yet, it will come with 2.1.0.

**Comment by muticansel** [comment]
ok, thank you

**Comment by djhi** [comment]
Closed by #1843
